src/storage/exec/QueryUtils.h

```diff
@@ -71,6 +71,10 @@ class QueryUtils final {
       }
       return Status::Error(folly::stringPrintf("Fail to read prop %s ", propName.c_str()));
     }
+    if (field->type() == meta::cpp2::PropertyType::FIXED_STRING) {
```

Comment:
I think you should remove the extra zero in the RowReader.

Reply:
> I think you should remove the extra zero in the RowReader.

No, there are other effects for reader. such as upgrade tool.

Reply:
Why upgrade tool need the raw value.